README.md

```diff
@@ -24,7 +24,7 @@ Not sure which edition is the right one for you? Check this handy table!
 | Add mods with few clicks | [✅](https://github.com/Madis0/fabulously-optimized/wiki/Adding-more-mods#curseforge-launcher-minecraft-117xfo-2xx) | [❌](https://github.com/Madis0/fabulously-optimized/wiki/Adding-more-mods#multimc) | [❌](https://github.com/Madis0/fabulously-optimized/wiki/Adding-more-mods#multimc) | [✅](https://github.com/Madis0/fabulously-optimized/wiki/Adding-more-mods#gdlauncher) | [❌](https://github.com/Madis0/fabulously-optimized/wiki/Adding-more-mods#minecraft-launcher-vanilla) | ❓ |
 | Can be used offline | ❌ | ✅ | ✅ | ✅ | ✅ | ❓ |
 | Supports mod developers | ✅ | ✅ | ❓ | ❓ | ✅ | ❓ |
-
+¹ Not implemented yet

```

Comment:
Vanilla is supported because I have the instructions for it, even though there is no installer yet.
Modrinth is not implemented indeed, but I think the X in the table indicates that good enough already.

Reply:
Got it, done. Feel free to cherry-pick my commits in order to keep a clean commit history.